docs/userdocs/k8s/api/experiment.md

```diff
@@ -167,6 +167,79 @@ Above example assume environment "my-submarine-env" already exists in Submarine.
 }
 ```
 
+## Create Experiment (Using Pre-defined/Stored Environment)
```

Comment:
I'm not sure why we make this change, I thought it will include /code init spec, but it didn't. Do you plan to add it later?

Reply:
This change shouldn't be part of this PR. It is part of https://github.com/apache/submarine/pull/348. Will check again.

Reply:
Taken care